docs/extensions.md

```diff
@@ -156,6 +156,54 @@ To get results from geocoding data for providers who aren't using Open Data when
 3. Enter your `Yelp API key` in your [application credentials](https://tripgo.3scale.net/admin/applications). 
 4. Add `allowYelp=true` to your geocoding requests, and then geocoding results will then come augmented with Yelp's API through the TripGo API for your API key (after at most 5 minutes).
 
+---
+
+## TSP connectors
+
+For TSPs that are not yet integrated into our platform, you can add them by:
+ 
+ - implementing the corresponding API (see the list of API specs available [here](https://skedgo.github.io/TSP-APIs/))
+ - sending to [api@skedgo.com](mailto:api@skedgo.com) the url to access that API
+ 
+The process then involves a manual step (which we plan to automate) to add that to our list of TSP connectors and enable it, and that's it, results will then include the new TSP.
```

Comment:
@MarianoTucat Can you add some information of what time delay? Does it really not need any way to enable these for an API key, or is that what you mean with "manual step"?

Reply:
Currently, we need to update the API keys, that may change in the future when we automate this step. BTW, does the bike lane info needs to be precomputed?